src/CommunityToolkit.Maui.MediaElement/Handlers/MediaElementHandler.shared.cs

```diff
@@ -19,7 +19,7 @@ public partial class MediaElementHandler
 	/// <summary>
 	/// The default property mapper for this handler.
 	/// </summary>
-	public static IPropertyMapper<MediaElement, MediaElementHandler> PropertyMapper = new PropertyMapper<MediaElement, MediaElementHandler>(ViewMapper)
+	public static readonly IPropertyMapper<MediaElement, MediaElementHandler> PropertyMapper = new PropertyMapper<MediaElement, MediaElementHandler>(ViewMapper)
```

Comment:
Even on .net maui the mappers aren't `readonly` as you can see [here](https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/Entry/EntryHandler.cs#L20). Probably, is intended to not be `readonly`, that way an user can override it.